t/154-semaphore.t

```diff
@@ -0,0 +1,115 @@
+# vim:set ft=ts=4 sw=4 et fdm=marker:
+use lib 'lib';
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(10140);
+#workers(1);
+#log_level('warn');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 3) + blocks();
+
+no_long_string();
+#no_diff();
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: timer + shutdown error log
+--- http_config
+    lua_package_path "../lua-resty-core/lib/?.lua;;";
```

Comment:
We should add `../lua-resty-lrucache/lib/?.lua` to the search paths here as well. We may also update other test cases (better use a Perl variable in the `http_config` section to avoid code duplication?